spec/lib/open_food_network/feature_toggle_spec.rb

```diff
@@ -1,30 +1,52 @@
 # frozen_string_literal: true
 
-require 'open_food_network/feature_toggle'
+require 'spec_helper'
 
 module OpenFoodNetwork
   describe FeatureToggle do
-    it "returns true when feature is on" do
-      stub_foo("true")
-      expect(FeatureToggle.enabled?(:foo)).to be true
-    end
+    context 'when users are not specifyied' do
```

Comment:
Typo :)
```suggestion
    context 'when users are not specified' do
```

Reply:
typos everywhere, probably too late for me :joy: